public/javascripts/accessScoreDemo.js

```diff
@@ -227,7 +227,7 @@ function initializeSubmittedLabels(map, southwest, northeast) {
             "stroke-width": 1
         };
 
-    $.getJSON(`/v2/access/attributes?lat1=${southwest.lat}&lng1=${southwest.lng}&lat2=${northeast.lat}&lng2=${northeast.lng}`, function (data) {
+    $.getJSON(`/v2/access/attributes?lat1=${southwest.lat}&lng1=${southwest.lng}&lat2=${northeast.lat}&lng2=${northeast.lng}&severity=${label.severity}`, function (data) {
```

Comment:
Why was this change made? I think it breaks the /demo endpoint b/c there is no `label` object defined.